<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopCarWindow</class>
 <widget class="QWidget" name="ShopCarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>购物车</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="table_shopcar">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_buy">
     <property name="text">
      <string>清空购物车~</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
